subsys/bluetooth/controller/ll_sw/ull_iso.c

```diff
@@ -30,12 +30,12 @@
 #include "lll_conn_iso.h"
 #include "lll_iso_tx.h"
 
-#if !defined(CONFIG_BT_LL_SW_LLCP_LEGACY)
-#include "ull_tx_queue.h"
-#endif
-
 #include "isoal.h"
 
+#if defined(CONFIG_BT_LL_SW_LLCP)
+#include "ull_tx_queue.h"
+#endif /* CONFIG_BT_LL_SW_LLCP */
```

Comment:
Ditto

Reply:
This include is only required for the new LLCP, and not for the legacy LLCP